### PR TITLE
Graphs: SVG and PNG for tests and docu.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

**Comment by dkorolev** [comment]
@deathbaba, @sompylasar PTAL.

File: graph/docu/docu_04graph_02plotutils.cc [hunk 1]
```diff
@@ -51,9 +56,15 @@ TEST(Graph, PlotutilsLove) {
     .X("... living life in peace")
     .Y("John Lennon, \"Imagine\"")
     .LineWidth(0.015)
-    .OutputFormat("png");
-
-ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString("golden/love.png"));
+    .BitmapSize(800, 800)
+#if 1
```
Comment (deathbaba) [inline graph/docu/docu_04graph_02plotutils.cc]: Do you really need it?

**Comment by deathbaba** [comment]
LGTM

File: graph/docu/docu_04graph_02plotutils.cc (hunk 1 above)
Comment (dkorolev) [inline graph/docu/docu_04graph_02plotutils.cc]: Yes for both the test to pass, for the code to generate a "png", and the documentation to look clean and complete.

**Comment by dkorolev** [comment]
Apparently, PNG-s are different day by day. Let me merge it this way and see if that holds up to tomorrow :)

**Pull request merged**